macolmenerori/opensesame-front
# [FEAT] Add navbar and logout functionality

### What's new

- New navbar
- New logout button

### What's fixed

Nothing fixed here...

### How to test

- Check out the new navbar
- Try the logout button

### Breaking changes

No breaking changes here...
